### PR TITLE
Fix image and file handling

### PR DESCRIPTION
**Pull request by carakas**

### CODE REVIEW

File: src/SumoCoders/FrameworkCoreBundle/ValueObject/AbstractFile.php [hunk 1]
```diff
@@ -250,6 +250,16 @@ public function setPendingDeletion($isPendingDeletion): void
         }
     }
 
+    /**
+     * @internal Used by the form types
+     *
+     * @return bool
+     */
+    public function isPendingDeletion()
```
Comment (ohvitorino) [inline src/SumoCoders/FrameworkCoreBundle/ValueObject/AbstractFile.php]: It is missing the return type `: bool`

File: src/SumoCoders/FrameworkCoreBundle/ValueObject/AbstractFile.php [hunk 2]
```diff
@@ -250,6 +250,16 @@ public function setPendingDeletion($isPendingDeletion): void
...
+    {
+        return \strlen($this->oldFileName) > 0 && $this->fileName === null;
```
Comment (ohvitorino) [inline src/SumoCoders/FrameworkCoreBundle/ValueObject/AbstractFile.php]: Does this need to be `\strlen`?

Comment (carakas) [reply src/SumoCoders/FrameworkCoreBundle/ValueObject/AbstractFile.php]: opcache optimalisations

File: src/SumoCoders/FrameworkCoreBundle/ValueObject/AbstractFile.php (hunk 1 above)
Comment (carakas) [reply src/SumoCoders/FrameworkCoreBundle/ValueObject/AbstractFile.php]: fixed in 
6f8a6f5

File: src/SumoCoders/FrameworkCoreBundle/ValueObject/AbstractFile.php (hunk 2 above)
Comment (ohvitorino) [reply src/SumoCoders/FrameworkCoreBundle/ValueObject/AbstractFile.php]: Care to share the docs of article mentioning this? :)

Comment (ohvitorino) [reply src/SumoCoders/FrameworkCoreBundle/ValueObject/AbstractFile.php]: I found something related to this:

https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3048

Comment (carakas) [reply src/SumoCoders/FrameworkCoreBundle/ValueObject/AbstractFile.php]: already shared it multiple times already :) it is not the first time I'm including it

Comment (jonasdekeukelaere) [reply src/SumoCoders/FrameworkCoreBundle/ValueObject/AbstractFile.php]: Don't really think it makes a difference in this use case, but ok

**Pull request merged**